src/Plugins/LevelDBStore/Options.cs

```diff
@@ -0,0 +1,213 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// Options.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+namespace LevelDB
+{
+    /// <summary>
+    /// Options to control the behavior of a database (passed to Open)
+    /// 
+    /// the setter methods for InfoLogger, Env, and Cache only "safe to clean up guarantee". Do not
+    /// use Option object if throws.
+    /// </summary>
+    public class Options : LevelDBHandle
```

Comment:
All to `internal`?

Reply:
We could but its a plugin.

Reply:
I thought about it. This is a `plugin` and its best to have it be `public`.